android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -401,8 +401,7 @@ public void call_getStats(String uuid,  Promise promise) {
 
   @ReactMethod
   public void call_sendMessage(String uuid, String content, String contentType, String messageType, Promise promise) {
```

Comment:
Thinking about it, is it strange to use `call_sendMessage` in _both_ `CallInvite` and `Call`? Do you think we should have different native methods for each object? How much repetition are we risking?

Reply:
This should not affect the user since we handle this function. However you are right as it is a little odd. @charliesantos any opinions?

Reply:
Since this is not a JS public API, I'm ok with either keeping it as is or use different native methods. However, if we end up using different native methods, we should not duplicate any code. Another option is, have two native methods (one for Call and one for CallInvite), and each native method calls another method to prevent duplication. Something like

public void call_sendMessage(...);
public void callInvite_sendMessage(...);
private void sendMessage(...);

Then call_sendMessage and callInvite_sendMessage will call sendMessage internally.

Reply:
For now lets keep it the current way. I see this somewhat similar to why we have a `callSid`, and not a `callInviteSid` for both a `callInvite` and `Call`